File: QuizApp/src/QuizApp.BLL/DTO/Test/NewTestDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QuizApp.BLL.DTO.Test
{
    public class NewTestDTO
    {
        public string Title { get; set; }

        public string Description { get; set; }

        public TimeSpan TimeLimitSeconds { get; set; }

        public int AuthorId { get; set; }
    }
}

File: frontend/src/app/models/test/new-test-dto.ts
import { NewQuestionDto } from '../question/new-question-dto';
import { NewUrlDto } from '../url/new-url-dto';

export interface NewTestDto {
    title: string;
    description: string;
    timeLimitSeconds: string;
    authorId: number;

    urls: NewUrlDto[];
    testQuestions: NewQuestionDto[];
}
